### PR TITLE
Anthony's own respository commit

### PR DESCRIPTION
**Pull request by acuen753**
My comment in the mainwindow is sooooo awesome!

### CODE REVIEW

**Comment by Iastre** [comment]
Is it really?

File: mainwindow.cpp [hunk 1]
```diff
@@ -14,6 +14,7 @@ MainWindow::~MainWindow()
 {
     delete ui;
 }
+//YAASSSSSSSS
```
Comment (Iastre) [inline mainwindow.cpp:17]: YAAASSS what?

**Comment by imora128** [comment]
HELLO?

**Comment by Iastre** [comment]
Another comment?

**Pull request closed without merging**